manager/state/store/networks.go

```diff
@@ -167,7 +172,7 @@ func GetNetwork(tx ReadTx, id string) *api.Network {
 func FindNetworks(tx ReadTx, by By) ([]*api.Network, error) {
 	checkType := func(by By) error {
 		switch by.(type) {
-		case byName, byNamePrefix, byIDPrefix:
+		case byName, byNamePrefix, byIDPrefix, byCustom, byCustomPrefix:
```

Comment:
Every object now supports custom indices using the `Indices` field in `Annotations`. I'm not sure offhand what use cases exist specifically for networks, but I think it's good to be consistent and support this for all objects.